### PR TITLE
feat(poap): implement transfer modification subscription and transfer poap

### PR DESCRIPTION
**Pull request by mthinh**
# What ?
- implement transfer modification subscription
- implement transfer poap

# Screenshot 

https://github.com/lemonadesocial/lemonade-flutter/assets/28992487/cdf1b244-2e5a-4db8-877d-8b7d389483e5

https://github.com/lemonadesocial/lemonade-flutter/assets/28992487/981ef41e-d57d-477e-a917-1a2cdfd79c21

### CODE REVIEW

**Review by roncoderpush — CHANGES_REQUESTED** [review]
Overall looking good

File: lib/core/presentation/widgets/poap/poap_transfer_success_poup/poap_transfer_success_popup.dart [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                child: Stack(
+                  children: [
+                    Align(
+                      alignment: Alignment.center,
+                      child: LayoutBuilder(
+                        builder: (context, constraints) => Transform.scale(
+                          scale: 1.8,
+                          child: RippleAnimation(
+                            size: constraints.maxWidth,
+                            color: const Color.fromRGBO(32, 41, 38, 1),
```
Comment (sieuhuflit) [inline lib/core/presentation/widgets/poap/poap_transfer_success_poup/poap_transfer_success_popup.dart:53]: Should define in Colors file

**Review by sieuhuflit — COMMENTED** [review]
Leaving some comment

File: lib/core/presentation/widgets/poap/poap_transfer_success_poup/poap_transfer_success_popup.dart [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          borderRadius: BorderRadius.circular(
+            LemonRadius.small,
+          ),
+          child: Stack(
+            children: [
+              Transform.translate(
+                offset: const Offset(0, -100),
+                child: Stack(
+                  children: [
+                    Align(
```
[the remark this answers is not in the document]
Comment (mthinh) [reply lib/core/presentation/widgets/poap/poap_transfer_success_poup/poap_transfer_success_popup.dart]: 👍

File: lib/core/presentation/widgets/poap/poap_transfer_popup/poap_transfer_popup.dart [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+              network: widget.token?.network ?? '',
+              address: widget.token?.contract ?? '',
+            ),
+          ),
+        );
+  }
+
+  void onTransferSuccess(Transfer transfer) {
+    Navigator.of(context).pop();
+  }
```
[the remark this answers is not in the document]
Comment (mthinh) [reply lib/core/presentation/widgets/poap/poap_transfer_popup/poap_transfer_popup.dart:96]: it's a popup, i think we wont need to use auto router

**Comment by mthinh** [comment]
@roncoderpush @sieuhuflit comment fixed, please help recheck

**Review by sieuhuflit — APPROVED** [review]
Look good, nice work 👍

**Pull request merged**